### PR TITLE
chore(site): Update for Helm Classic

### PR DESCRIPTION
**Pull request by krancour**
This updates the site for the upcoming Helm Classic release.  Even once LGTM'ed, this should not be merged until Helm Classic 0.7.0 is released.

cc @slack @jackfrancis 

@jackfrancis Front end stuff is not my forte.  Hopefully I did alright, but I'm banking on you schooling me if I made any major missteps.

### CODE REVIEW

**Comment by krancour** [comment]
Fails CI on a 404 from bintray during validation.  This is because a stable helmc has not been released to there yet.  One we release Helm Classic v0.7.0, this problem will go away and the PR should pass CI.

File: _layouts/index.html [hunk 1]
```diff
@@ -18,32 +18,53 @@ <h1 class="logo">Helm</h1>
...
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, since so many in the Kubernetes community have come to depend on the
+              original Helm tool, the Deis team is committed to maintaining that original tool as
+              <i>Helm Classic</i>-- new name, same great taste! The name change ensures the original
+              and next generation Helm binaries can exist side-by-side and that the two tools can be
+              discussed unambiguously.
+            </p>
+            <p>
+              Read up on Helm Classic below and stay tuned for updates on
```
Comment (slack) [inline _layouts/index.html]: I'd just shorten this to "For more information and updates about the new Helm check out github.com/kubernetes/helm".

File: _layouts/index.html [hunk 2]
```diff
@@ -18,32 +18,53 @@ <h1 class="logo">Helm</h1>
...
     </section>
 
+    <section class="panel -blue" data-purpose="news-and-updates">
+      <div class="news-and-updates">
+        <div class="news-and-updates__item container group">
+          <h2>Helm comes in two great flavors!</h2>
+          <article class="console">
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, since so many in the Kubernetes community have come to depend on the
```
Comment (slack) [inline _layouts/index.html]: Style comment, so as you please. I'd shorten this to "The Deis team is committed to maintaining the original tool as 'Helm Classic' until the new Helm serves as a viable replacement--new name, same great taste!".

File: _layouts/index.html [hunk 3]
```diff
@@ -18,32 +18,52 @@ <h1 class="logo">Helm</h1>
...
 
+    <section class="panel -blue" data-purpose="news-and-updates">
+      <div class="news-and-updates">
+        <div class="news-and-updates__item container group">
+          <h2>Helm comes in two great flavors!</h2>
+          <article class="console">
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, the Deis team is committed to maintaining the original tool as
+              <i>Helm Classic</i>-- new name, same great taste! The name change ensures the original
```
Comment (jackfrancis) [inline _layouts/index.html]: For maximum æsthetic foo, I'd go with the HTML entity `&mdash;` instead of `--`. And surround the em dash with a single space on each side, e.g.,

`<i>Helm Classic</i> &mdash; new name`...

**Comment by jackfrancis** [comment]
Added one suggestion, otherwise LGTM pending CI!

File: _layouts/index.html (hunk 3 above)
Comment (krancour) [inline _layouts/index.html]: Great.  I will take you up on that suggestion!  Thanks!
